neuraxle/hyperparams/distributions.py

```diff
@@ -246,7 +254,7 @@ def __init__(self, choice_list: List):
 
         :param choice_list: a list of values to sample from.
         """
-        HyperparameterDistribution.__init__(self)
+        HyperparameterDistribution.__init__(self, None)
```

Comment:
Also edit the return value of `narrow_space_from_best_guess` for this current `Choice` class.

Reply:
@guillaume-chevalier ?

Reply:
Also why do you have just `, None` here?